components/BundleModal.tsx

```diff
@@ -190,23 +190,26 @@ function BundleTransaction(transaction, index: number) {
   };
 
   useEffect( () => {
-    const getLogs = async () => setLogs(await getReceipts(transaction));
+    const getLogs = async () => {
+      const _logs = await getReceipts(transaction);
+      let _coins = [];
```

Comment:
why destructuring the `reduce`?

Reply:
change to map becuz reduce was throwing error